ocaml/testsuite/tests/typing-layouts-float64/alloc.ml

```diff
@@ -0,0 +1,90 @@
+(* TEST
+   flags = "-extension layouts_alpha"
+   * native
+*)
+
+(* A test comparing allocations with unboxed floats to allocations with boxed
+   floats. *)
+
+(* CR layouts v2: test shouldn't print exact allocation numbers. And I've only
```

Comment:
I agree here, but hopefully it doesn't allocate at all on any of the middle ends.

Reply:
(I was more worried about whether the boxed version also wouldn't allocate on the other middle ends.  After testing, `flambda2` with `-O3` does manage to get to 0 allocations, so I have added a `[@inline never]` in an appropriate place to foil it.)